src/coord/timestamp.rs

```diff
@@ -1299,7 +1298,18 @@ impl Timestamper {
             .set("enable.sparse.connections", "true")
             .set("bootstrap.servers", &kc.url.to_string());
 
-        for (k, v) in &kc.config_options {
+        let mut config_options = kc.config_options.clone();
+
+        let group_id_prefix = match config_options.remove("group.id.prefix") {
+            Some(supplied_prefix) => supplied_prefix,
+            None => "".to_string(),
+        };
+        config.set(
+            "group.id",
+            &format!("{}materialize-rt-{}-{}", group_id_prefix, &kc.topic, id),
```

Comment:
Should we format as `PREFIX-materialize...` when the prefix is present? That's how sink prefixes work at the moment, FWIW.